javascript/sponsor.js

```diff
@@ -0,0 +1,15 @@
+const sponsorLink = document.getElementsByClassName('sponsor-nav-link')[0]
+
+sponsorLink.addEventListener('mouseover', scaleDecreaseHeart)
+
+sponsorLink.addEventListener('mouseout',  scaleIncreaseHeart)
+
+function scaleIncreaseHeart() {
+  let svg = sponsorLink.getElementsByTagName('svg')[0];
+  svg.setAttribute('transform', `scale(1)`);
+}
+
+function scaleDecreaseHeart(){
+  let svg = sponsorLink.getElementsByTagName('svg')[0];
+  svg.setAttribute('transform', `scale(1.2)`);
+}
```

Comment:
Is this done because we can't use `:hover` CSS?

Reply:
I use this idea because when you have this element, like `a/button`, and you need to scale only the heart when the entire element area has hovered, only adding events is possible otherwise it will scale the entire element.

Reply:
Sorry, perhaps I'm getting confused. Wouldn't a selector like `a:hover svg` work here?